### DISCUSSION TITLE
Anyone have a stripe payment form with shadcn components

### DISCUSSION THREAD

**Discussion by Elaminebendaas** [shadcn-ui/ui#3806] [Q&A]
I am looking to integrate stripe in to my website instead of it redirecting to stripe I set it to allow users to do payments directly on the site without redirects.

Anyone know of a shadcn template/block for payments

**Comment by 21d3v** [comment]
found this: https://github.com/badiniibrahim/nextjs-stripe-integration/blob/main/README.md
